app/assets/javascripts/miq_tree.js

```diff
@@ -394,21 +420,21 @@ function miqSquashToggle(treeName) {
 
 function miqTreeEventSafeEval(func) {
   var whitelist = [
-    'miqOnCheckCUFilters',
-    'miqOnCheckGenealogy',
+    'miqOnCheckCUFilters', // Settings -> Region -> CU Collection tab
+    'miqOnCheckGenealogy', // Compute -> Ingrastructure -> VMs -> Select one vm and click on genealogy
     'miqOnCheckGeneric',
-    'miqOnClickMenuRoles',
-    'miqOnCheckProtect',
-    'miqOnCheckSections',
+    'miqOnClickMenuRoles', // Settings -> Access Controll -> Roles/Edit Roles // Seems like not doing anything
+    'miqOnCheckProtect', // Compute -> Infra -> VMs -> Select VM -> Policy button -> edit
+    'miqOnCheckSections', // Compute -> Infra -> VMs -> Compare VM's
     'miqOnCheckUserFilters',
-    'miqOnClickAutomate',
-    'miqOnClickAutomateCatalog',
-    'miqOnClickDiagnostics',
-    'miqOnClickGeneric',
-    'miqOnClickHostNet',
-    'miqOnClickSnapshots',
-    'miqOnClickUtilization',
-    'miqOnCheckTenantTree',
+    'miqOnClickAutomate', // Automate -> Expoler -> Select a class -> Copy -> Uncheck Copy to same path -> Namespace selection uses thath tree
+    'miqOnClickAutomateCatalog', // Services -> Catalogs -> Catalog Items -> add a new item -> select Generic -> three entry point fields open it.
+    'miqOnClickDiagnostics', // Settings -> Diagnostics -> Roles By servers tab
+    'miqOnClickGeneric', // Compute -> Ingrastructure -> VMs -> Select one vm and click on genealogy
+    'miqOnClickHostNet', // Compute -> Infra -> Networking // Still old tree
+    'miqOnClickSnapshots', // Compute -> Infra -> VMs -> Select one -> Snapshot button
+    'miqOnClickUtilization', // Non existent
+    'miqOnCheckTenantTree', // Still angular component
```

Comment:
It's no longer an angular component, make the comment consistent with the rest of it.